Examples/Standups/Standups/AppFeature.swift

```diff
@@ -25,7 +25,7 @@ struct AppFeature: Reducer {
     Scope(state: \.standupsList, action: /Action.standupsList) {
       StandupsList()
     }
-    Reduce<State, Action> { state, action in
```

Comment:
Should this be removed if it means that Xcode now has a very hard time auto-completing the action enums? Looks like it's still a problem in Xcode 15 and Swift 5.9. I don't have a problem with it; just keeping in mind in case this was forgotten :)

Reply:
You're right; my apologies. I am unable to reproduce with the reducers within your codebase. I am only able to reproduce in my reducers that have tons of cases (since I'm utilizing ViewAction, DelegateAction, InternalAction enums) but when I clean up a lot of it; the auto-complete is correct.